internal/pritunl/route.go

```diff
@@ -41,6 +41,9 @@ func ConvertMapToRoute(data map[string]interface{}) Route {
 	if v, ok := data["nat"]; ok {
 		route.Nat = v.(bool)
 	}
+	if v, ok := data["net_gateway"]; ok {
+		route.NetGateway = v.(bool)
```

Comment:
I see the setter of the `NetGateway` property but didn't see any place where you use this property in the provider. Do you need just to store this property in the state or I miss something?

Reply:
We need to create net gateway route, would this change set the route type when it invokes the api to create the route?

Reply:
I found some places where it could be used,  would these changes work, please suggest